pagopa/io-app-design-system
# [SIW-942]  Add `payments` pictograms

## Short description
This PR introduces the `payments` pictograms.

<img src="https://github.com/pagopa/io-app-design-system/assets/6160324/6615559c-f759-4e92-8de9-935c49e33ce5" width="150" />
<img src="https://github.com/pagopa/io-app-design-system/assets/6160324/86ce3ac5-2d25-4c0d-91f3-5a811c415143" width="150" />


## List of changes proposed in this pull request
- Added `PictogramPayments`
- Added `PictogramBleedPayments`
- Updated `IOPictograms` and `IOPictogramsBleed`

## How to test
Within the example app, navigate to the **Pictograms** section and check that the new pictograms are correctly rendered
